playbooks/compute/delete_old_clusters.yml

```diff
@@ -0,0 +1,72 @@
+- name: Deploy NTO configuration
+  hosts: hypervisor
+  gather_facts: false
+  become: true
+  vars:
+    libvirt_storage_path: /var/lib/libvirt/images
+
+  tasks:
+    - name: Get all Running VMs
+      community.libvirt.virt:
+        command: list_vms
+      register: running_vms_result
+
+    - name: Filter VMs with "master" or "worker" in the name
+      ansible.builtin.set_fact:
+        vms_to_delete: "{{ running_vms_result.list_vms | select('match', '(?i).*_(master|worker)[0-9].*') | list }}"
```

Comment:
what happens if the list_vms is empty here?

Reply:
nothing - there is no loop if the list is empty 
but you should have the bastion and/or executor